package/osx/CMakeLists.txt

```diff
@@ -3,19 +3,19 @@
 if (RSTUDIO_PACKAGE_BUILD)
    INSTALL(CODE "
      list (APPEND CODESIGN_TARGETS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app\")
-  
+
      file(GLOB_RECURSE CODESIGN_PLUGINS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app/Contents/plugins\")
      list (APPEND CODESIGN_TARGETS \${CODESIGN_PLUGINS})
 
      file(GLOB_RECURSE CODESIGN_FRAMEWORKS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app/Contents/Frameworks\")
      list (APPEND CODESIGN_TARGETS \${CODESIGN_FRAMEWORKS})
-  
+
      file(GLOB_RECURSE CODESIGN_MACOS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app/Contents/MacOS\")
      list (APPEND CODESIGN_TARGETS \${CODESIGN_MACOS})
-  
+
      foreach(CODESIGN_TARGET \${CODESIGN_TARGETS})
-        execute_process(COMMAND codesign \"-s\" \"Developer ID Application: RStudio Inc. (FYF2F5GFX4)\" \"--deep\" \"-i\" \"org.rstudio.RStudio\" \"\${CODESIGN_TARGET}\")
-     endforeach()   
+        execute_process(COMMAND codesign \"-s\" \"8A388E005EF927A09B952C6E71B0E8F2F467AB26\" \"--deep\" \"-i\" \"org.rstudio.RStudio\" \"\${CODESIGN_TARGET}\")
```

Comment:
Is this another way of specifying the same cert? (want to make sure this will continue to work on our existing native build as well as the vagrant image)